tools/build_virtual_environment/dynamic_ve_base/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM dart:3.6.1@sha256:a071f0322c7c80469842f7c8258de51d1946ba9b79d09bb8fc4dc969487a68ca AS buildimage
+ENV USER_ID=1024
+ENV GROUP_ID=1024
+WORKDIR /app
+# Context for this Dockerfile needs to be at_server repo root
+# If building manually then (from the repo root):
+## sudo docker build -t atsigncompany/dynamicvebase \
+## -f tools/build_virtual_environment/dynamic_ve_base/Dockerfile .
+COPY . .
+RUN \
+  cd /app/packages/at_root_server ; \
+  dart pub get ; \
+  dart pub update ; \
+  dart compile exe bin/main.dart -o root ;
+# Disabled for dynamic_ve_base
+# cd /app/tools/build_virtual_environment/install_PKAM_Keys ; \
```

Comment:
Could we avoid a lot of duplication by having a Dockerfile like this as the new VE base (for both variants) and then have another base image that we build on top of that with this stuff as an extra layer?

Reply:
Sure, I can do that. I've put this PR as an item in the next sprint for me to make these changes. For now, this is enough to continue onwards with the demo I'm working on.